parsers/US_ERCOT.py

```diff
@@ -35,7 +35,7 @@
     "Natural Gas": "gas",
     "Nuclear": "nuclear",
     "Other": "unknown",
-    "Power Storage": "battery",
+    "Power Storage": "unknown",  # we lose this information in the EIA parser and have no easy way to split it when we run fetch_production for past dates. As it is a minor share of the production mix, it will be categorized as unknown
```

Comment:
Just to be clear here, is this always positive or do we need to make sure it's not negative so it reduces the unknown production? (or how does EIA do it)

Reply:
looks like they don't account for the storage part and only what is injected on the mix